crates/red_knot_python_semantic/resources/mdtest/binary/booleans.md

```diff
@@ -0,0 +1,25 @@
+## Binary operations on booleans
+
+## Basic Arithmetic
+
+```py
+a = True
+b = True
+c = a + b
+d = a - b
+e = a * b
+f = a % b
+g = a // b
+h = a ** b
+i = a / b
+j = a | b
+
+reveal_type(c)  # revealed: int
+reveal_type(d)  # revealed: int
+reveal_type(e)  # revealed: int
+reveal_type(f)  # revealed: int
+reveal_type(g)  # revealed: int
+reveal_type(h)  # revealed: int
+reveal_type(i)  # revealed: float
+reveal_type(j)  # revealed: bool
```

Comment:
nit: this pattern of creating many variables is a legacy holdover from our previous testing infrastructure, which was more limited in the assertions it could make. For cases like this with small expressions and small types, it's more readable to just do e.g. `reveal_type(a + b)  # revealed: int` all in one line. (In fact, a PR changing existing tests to use this style would also be welcome)

Reply:
> a PR changing existing tests to use this style would also be welcome

I had already started with this, so here it is: https://github.com/astral-sh/ruff/pull/13806